sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -460,7 +460,12 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
               // because `distinctExpressions` is not extracted during logical phase.
               NormalizeFloatingNumbers.normalize(e) match {
                 case ne: NamedExpression => ne
-                case other => Alias(other, other.toString)()
```

Comment:
@cloud-fan, what about other places we call `NormalizeFloatingNumbers.normalize(e)`, for example above?

Reply:
`groupingExpressions` is `Seq[NamedExpression]`, so it already preserves the original name.

Reply:
I see, thanks. Can we at least leave a short comment why we're doing it here alone?